docs/how-to-guides/control-the-case-of-identifiers.md

```diff
@@ -0,0 +1,134 @@
+# How to control the case of identifiers
+
+## Context
+
+You have identifiers, like row names, that will be used outside of
+ksqlDB. You want to control the exact casing (capitalization) of how
+they are represented to make them consumable by downstream
+programs. Because ksqlDB uppercases all identifiers by default, you
+need to use backticks to preserve the desired casing.
+
+## In action
+
+```sql
+CREATE STREAM `s1` (
+    `foo` VARCHAR KEY,
+    `BAR` INT,
+    `Baz` VARCHAR
+) WITH (
+    kafka_topic = 's1',
+    partitions = 1,
+    value_format = 'avro'
+);
+```
+
+## Backticks
+
+Begin by telling ksqlDB to start all queries from the earliest point
+in each topic.
+
+```sql
+SET 'auto.offset.reset' = 'earliest';
+```
+
+Declare a new stream named `s2`. In this example, you override
+ksqlDB's default behavior to uppercase all identifiers. Use backticks
+to control the casing of the stream name and column names. To contrast
+this behavior, `quz` is declared without backticks to demonstrate
+the default behavior of uppercasing.
+
+```sql
+CREATE STREAM `s2` (
+    `foo` VARCHAR KEY,
+    `BAR` INT,
+    `Baz` VARCHAR,
+    `grault` STRUCT<
+        `Corge` VARCHAR,
+        `garply` INT
```

Comment:
out of interest, where did you come up with these? 😂

Reply:
elementary, my dear Watson: https://en.wikipedia.org/wiki/Metasyntactic_variable

Reply:
we should add `franz` to our standard canon of metasyntactic variables

Reply:
Hah, you found it. :) I ran out and googled "foo bar baz qux" and clicked the first link. Glad there's more, my choices would have been terrible. 😂